bears/elm/ElmLintBear.py

```diff
@@ -0,0 +1,37 @@
+from shutil import which
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='elm-format',
+        use_stdin=True,
```

Comment:
We have `output_format = regex` and `output_format = corrected`. Since it only detects the error, I think `corrected` cannot be used and since we don't want to use regex, should we go for overriding `process_output` ? @Makman2